### ISSUE TITLE
Cannot genenerate Bower config using spago

### ISSUE THREAD

**Issue by thautwarm** [purescript/spago#578]
```
bash>spago bump-version --no-dry-run v0.1.0
[info] Generating a new Bower config using the package set versions..
INFO: Could not find files for the given pattern(s).
````

The package is [purescript-show-python](https://github.com/purescript-python/purescript-show-python
).
I've already read some configs from repos of published libraries, such as [purescript-dotenv](https://github.com/nsaunders/purescript-dotenv). Just cannot figure out what's wrong with me.

**Comment by f-f** [comment]
@thautwarm are you on the latest Spago version? That command works for me, and I get this `bower.json`:
```
{
    "name": "purescript-show-python",
    "license": [
        "MIT"
    ],
    "repository": {
        "type": "git",
        "url": "https://github.com/purescript-python/purescript-show-python.git"
    },
    "ignore": [
        "**/.*",
        "node_modules",
        "bower_components",
        "output"
    ],
    "dependencies": {
        "purescript-effect": "^v2.0.1",
        "purescript-prelude": "^v4.1.1"
    }
}

```

**Comment by thautwarm** [comment]
@f-f I found this is platform-specific. Above error just occurs on Windows, but in any Linux distros(arch and ubuntu) things are totally fine.

**Comment by thautwarm** [comment]
I've already use `npm update -g spago` in Windows.

**Comment by thautwarm** [comment]
Got it. In Windows I didn't install `bower`, and the error message tells just nothing.
In Linux, the error message is 
```
spago: bower: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
```
which makes a lot of sense and can drive me to accomplish the following steps.

I'll close this issue, but I'd strongly suggest to make a better error reporting for Windows users.

**Comment by f-f** [comment]
@thautwarm yes, this makes sense. I fixed this in #610, where we look for the bower executable beforehand and fail with a decent error message if not found.
